client/components/Help/HelpPortal.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect, FC } from 'react'
+import { Modal, ModalHeader, ModalBody } from 'reactstrap'
+import Crowi from 'client/util/Crowi'
+
+interface Props {
+  crowi: Crowi
+}
+
+const HelpPortal: FC<Props> = ({ crowi }) => {
+  const [show, setShow] = useState(false)
+
+  const handleOpen = () => {
+    setShow(true)
+  }
+
+  const handleClose = () => {
+    setShow(false)
+  }
+
+  useEffect(() => {
+    $("a[data-target='#help-portal']").on('click', handleOpen)
```

Comment:
Please add the dependent value(`handleOpen`) to the second argument.

Reply:
done.